automation/taskcluster/androidTest/flank-x86.yml

```diff
@@ -23,7 +23,7 @@ gcloud:
 
   # The number of times a TestExecution should be re-attempted if one or more\nof its test cases fail for any reason.
   # The maximum number of reruns allowed is 10. Default is 0, which implies no reruns.
-  num-flaky-test-attempts: 2
+  num-flaky-test-attempts: 0
```

Comment:
Let's just remove the parameter altogether.  This is an optional param that would be 0 anyway if left off.  I'll approve for now since just simple nit.  Let's get this landed as soon as green 👍